src/Symfony/Component/Notifier/Bridge/GoogleChat/GoogleChatTransportFactory.php

```diff
@@ -37,11 +38,20 @@ public function create(Dsn $dsn): TransportInterface
         $space = explode('/', $dsn->getPath())[1];
         $accessKey = $this->getUser($dsn);
         $accessToken = $this->getPassword($dsn);
-        $threadKey = $dsn->getOption('threadKey');
+
+        $threadKey = $dsn->getOption('thread_key');
+
+        /*
+         * Remove this check for 5.4
```

Comment:
The bridge were experimental in previous versions. Do we need to keep a BC layer?

Reply:
We did the same for Slack, so this doesn't hurts and brings a good DX